### PR TITLE
[#44052] rework of file list style overwrites

### PR DESCRIPTION
**Pull request by Kharonus**
- https://community.openproject.org/work_packages/44052
- amended spot list flex alignment
- reduced spot list indention selector specifity

### CODE REVIEW

File: frontend/src/global_styles/content/work_packages/tabs/_files.sass [hunk 1]
```diff
@@ -88,6 +89,10 @@
         &-icon
           margin-right: $spot-spacing-0-5
 
+  &--file-list &--file-list-item &--file-list-item-action
```
Comment (b12f) [inline frontend/src/global_styles/content/work_packages/tabs/_files.sass:92]: ;_;

File: frontend/src/global_styles/content/work_packages/tabs/_files.sass [hunk 2]
```diff
@@ -61,13 +59,16 @@
       &-text
         @include spot-caption()
 
+        line-height: 1.5rem
```
Comment (b12f) [inline frontend/src/global_styles/content/work_packages/tabs/_files.sass:62]: Can you do this as a `spot-spacing` too?

File: frontend/src/global_styles/content/work_packages/tabs/_files.sass (hunk 1 above)
Comment (b12f) [reply frontend/src/global_styles/content/work_packages/tabs/_files.sass:92]: (this is not a change request)

File: frontend/src/global_styles/content/work_packages/tabs/_files.sass (hunk 2 above)
Comment (Kharonus) [reply frontend/src/global_styles/content/work_packages/tabs/_files.sass:62]: Henriette told me, that I shouldn't use "spacing" for font sizes and line heights. Do you have a different opinion?

File: frontend/src/global_styles/content/work_packages/tabs/_files.sass (hunk 1 above)
Comment (Kharonus) [reply frontend/src/global_styles/content/work_packages/tabs/_files.sass:92]: yeah, found no other way of putting those padding overwrites.

**Review by b12f — APPROVED** [review]
LGTM

**Pull request merged**